Content.Server/Chemistry/Metabolism/MovespeedModifierMetabolism.cs

```diff
@@ -0,0 +1,61 @@
+using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Content.Server.GameObjects.Components.Chemistry;
+using System.Threading;
+using Content.Shared.Chemistry.Metabolizable;
+using Content.Shared.Movement.Components;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Default metabolism for stimulants and tranqs. Attempts to find a MovementSpeedModifier on the target,
+    /// adding one if not there and to change the movespeed
```

Comment:
this doccomment seems inaccurate to what this metabolism actually does, it doesn't add a movementspeedmodifier component anywhere, but a movement speed status comp